### ISSUE TITLE
Possible improvements on v0.7.0

### ISSUE THREAD

**Issue by leifdenby** [convml/convml-tt#21]
- [x] improve robustness of classification by using [shift-invariant convolutions](https://richzhang.github.io/antialiased-cnns/)
- [ ] implement purely convolutional architecture by removing adaptive-pooling and fully connected layers in architecture "head"
- [ ] improving speed by applying augmentation transforms per-batch and normalization transforms before creating memory-mapped array
- [x] one-cycle learning rate scheduler (gives order of magnitude speed-up on training), this is built into fastai: https://github.com/dkumazaw/onecyclelr
- investigate impact of:
    - [ ] changing number of embedding dimensions
    - [ ] changing batch-size
    - [ ] using model with random initial weights rather than pre-trained network
    - [ ] using deeper network (larger than resnet34)
